### DISCUSSION TITLE
gcc版本是否有限制

### DISCUSSION THREAD

**Discussion by syhien** [liusy58/NJU_AdvancedProgramming_SP22#156] [Q&A] [answered]
1. 用了c++20的 Range ，但是需要gcc 10，我可以自行安装gcc 10并在Makefile里用吗？
2. 如果可以，我需要在Makefile里面准备好gcc 10吗（意思就是会不会需要在我的机器以外验收

**Comment by RicoloveFeng** [comment, marked as answer]
验收是在你的电脑上验收，所以环境自己配好就行了
